Makefile

```diff
@@ -52,8 +51,7 @@ IMPORT_OPTIONS := --vendor \
 		--registry-url=leader.telekube.local:5000 \
 		$(IMPORT_IMAGE_OPTIONS)
 
-TELE_BUILD_OPTIONS := --insecure \
-		--repository=$(OPS_URL) \
+TELE_BUILD_OPTIONS := --repository=$(OPS_URL) \
```

Comment:
Why did you delete `--insecure` flag here?

Reply:
Good question, IMO it should be passed as `EXTERNAL_GRAVITY_OPTIONS` or every command should have `--insecure`. If not we're passing `--insecure` with `EXTERNAL_GRAVITY_OPTIONS` for _some_ commands and using an implicit `--insecure` for others and we have to use different env variables to don't pass `--insecure` twice.